### DISCUSSION TITLE
BinderException when running multi-argument count

### DISCUSSION THREAD

**Discussion by djouallah** [duckdb/duckdb#10364] [Ideas]
### What happens?

the same query works fine for spark SQL

### To Reproduce

```sql
create table summary_duck (duid int, settlementdate datetime);
select count(DUID, SETTLEMENTDATE) from summary_duck;
```

I get this error not sure what do with it
```
BinderException: Binder Error: No function matches the given name and argument types 'count(VARCHAR, TIMESTAMP)'. You might need to add explicit type casts.
	Candidate functions:
	count(ANY) -> BIGINT
	count() -> BIGINT
```

### OS:

Linux

### DuckDB Version:

0.9.3.dev2938

### DuckDB Client:

Python

### Full Name:

mim

### Affiliation:

personal

### Have you tried this on the latest `main` branch?

I have tested with a main build

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] Yes, I have

**Comment by szarnyasg** [comment]
Hello @djouallah, I edited your title/post to make the example reproducible and minimized it in the process.

Currently, we only support a single argument for `count`. The query does not work in Postgres either:

```
postgres=# select count(DUID, SETTLEMENTDATE) from summary_duck;
ERROR:  function count(integer, timestamp without time zone) does not exist
```

You could work around this by using a list:
```
select count(distinct [DUID, SETTLEMENTDATE]) from summary_duck;
```

**Comment by djouallah** [comment]
can you please change it to a feature request then just for the sake of compatibility with Spark SQL ?

**Comment by szarnyasg** [comment]
Sure
